UnityProject/Assets/Scripts/Systems/Botany/PlantData.cs

```diff
@@ -205,87 +206,76 @@ public void Mutation()
 		/// <param name="modification"></param>
 		public void NaturalMutation(PlantTrayModification modification)
 		{
-			//Chance to actually mutate
-			//if (random.Next(1, 2) != 1) return;
-
 			//Stat mutations
-			WeedResistance = StatMutation(WeedResistance, 100);
-			WeedGrowthRate = BadStatMutation(WeedGrowthRate, 100);
-			GrowthSpeed = StatMutation(GrowthSpeed, 100);
-			Potency = StatMutation(Potency, 100);
-			Endurance = StatMutation(Endurance, 100);
-			Yield = StatMutation(Yield, 100);
-			Lifespan = StatMutation(Lifespan, 100);
+			WeedResistance = StatMutation(WeedResistance, StatMutationType.Normal);
+			WeedGrowthRate = StatMutation(WeedGrowthRate, StatMutationType.Bad);
+			GrowthSpeed = StatMutation(GrowthSpeed, StatMutationType.Normal);
+			Potency = StatMutation(Potency, StatMutationType.Normal);
+			Endurance = StatMutation(Endurance, StatMutationType.Normal);
+			Yield = StatMutation(Yield, StatMutationType.Normal);
+			Lifespan = StatMutation(Lifespan, StatMutationType.Normal);
 			switch (modification)
 			{
 				case PlantTrayModification.None:
 					break;
 				case PlantTrayModification.WeedResistance:
-					WeedResistance = SpecialStatMutation(WeedResistance, 100);
+					WeedResistance = StatMutation(WeedResistance, StatMutationType.Special);
 					break;
 				case PlantTrayModification.WeedGrowthRate:
-					WeedGrowthRate = SpecialStatMutation(WeedGrowthRate, 100);
+					WeedGrowthRate = StatMutation(WeedGrowthRate, StatMutationType.Special);
 					break;
 				case PlantTrayModification.GrowthSpeed:
-					GrowthSpeed = SpecialStatMutation(GrowthSpeed, 100);
+					GrowthSpeed = StatMutation(GrowthSpeed, StatMutationType.Special);
 					break;
 				case PlantTrayModification.Potency:
-					Potency = SpecialStatMutation(Potency, 100);
+					Potency = StatMutation(Potency, StatMutationType.Special);
 					break;
 				case PlantTrayModification.Endurance:
-					Endurance = SpecialStatMutation(Endurance, 100);
+					Endurance = StatMutation(Endurance, StatMutationType.Special);
 					break;
 				case PlantTrayModification.Yield:
-					Yield = SpecialStatMutation(Yield, 100);
+					Yield = StatMutation(Yield, StatMutationType.Special);
 					break;
 				case PlantTrayModification.Lifespan:
-					Lifespan = SpecialStatMutation(Lifespan, 100);
+					Lifespan = StatMutation(Lifespan, StatMutationType.Special);
 					break;
 			}
 
-			CheckMutation(WeedResistance, 0, 100);
-			CheckMutation(WeedGrowthRate, 0, 100);
-			CheckMutation(GrowthSpeed, 0, 100);
-			CheckMutation(Potency, 0, 100);
-			CheckMutation(Endurance, 0, 100);
-			CheckMutation(Yield, 0, 100);
```

Comment:
the problem is with some of these that they're not supposed to go to hundred but look like someone accidentally set them to go to hundred, should this be included as part of PR?
![image](https://user-images.githubusercontent.com/27101670/129398325-e85e22d4-136b-47f3-8900-c2bea5ecce35.png)

Reply:
tho idk if they changed the corresponding code to match the new scheme